walkthroughs/http-retry-policy/README.md

```diff
@@ -0,0 +1,144 @@
+# HTTP Retry Policy example
+
+1. Clone this repository and navigate to the walkthrough/http-retry-policy folder, all commands will be ran from this location
+
+
+## Set Environment Variables
+
+1. **Your** account id:
+    ```
+    export AWS_ACCOUNT_ID=<your_account_id>
+    ```
+
+2. Preview is only in us-west-2
+    ```
+    export AWS_DEFAULT_REGION=us-west-2
+    ```
+
+## Create a VPC
+This is the virtual network for everything to run in. It is *pretty much* just the VPC from our official colorapp demo on github. It supports DNS, has a couple public and private subnets (1 per AZ), and an internet gateway so we can reach it and it can reach EMS. You can check it out in `vpc.yaml`:
+```
+aws cloudformation create-stack --stack-name retrypolicy-vpc --template-body file://vpc.yaml
+```
+And check the status until it is completed:
+```
+aws cloudformation describe-stacks --stack-name retrypolicy-vpc
+```
+
+## Upload Apps to ECR
+
+1. First we'll need to create 2 ECR repositories for our apps:
+    ```
+    aws ecr create-repository --repository-name retrypolicy-ecr-colorapp
+    ```
+    ```
+    aws ecr create-repository --repository-name retrypolicy-ecr-feapp
+    ```
+2. Build the colorapp Docker image: a http server that just returns the value of its `COLOR` environment variable. This command also tags the image with the ECR repo we created earlier:
```

Comment:
Could we modify this statement to explain how the colorapp is "special"? Specifically how the 'test-header' operates and controls HTTP response codes.